pytest.sh

```diff
@@ -1,6 +1,4 @@
 export P_HOST
-psql -c 'CREATE ROLE mecha3 LOGIN CREATEDB' -U postgres -h $P_HOST
```

Comment:
do we really need a shell script here?

Reply:
not anymore, testing moved outside docker